frontend/components/course/CourseTitleSubTitleBlock.tsx

```diff
@@ -16,7 +16,6 @@ export const CourseTitleSubTitleBlock: FC<IProps> = ({ course }) => {
     <div className="flex flex-1 flex-col">
       <span className="text-xs">
         <CourseWeekday course={course} /> <CourseStartTime course={course} />
-        {" - "}
         <CourseEndTime course={course} />
```

Comment:
It's a little ambiguous: Between `<it's a little ambiguous />` and `< CourseStartTime />` we need to render a whitespace. `< CourseEndTime />` has it's one whitespace prefix. Adding a whitespace between `<CourseStartTime />` and `<CourseEndTime />` would result in two whitespaces.

Reply:
Yeah, you are right. In the end I didn't like the string logic to be inside components and I now created helper functions instead of components. This also made it more obvious where to put spaces and there are no implicit spaces somewhere in the middle of JSX where no one expects them.